### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by henocookie**
Updated the README.md file to include up-to-date environment information.

### CODE REVIEW

**Comment by henocookie** [comment]
@DomBaker I know you mentioned the process to release to Live isn't quite the same as the current guidance. Not sure what the current process is for releasing to Live, do push any changes to this branch which would reflect the current process better 😀

**Comment by DomBaker** [comment]
> @DomBaker I know you mentioned the process to release to Live isn't quite the same as the current guidance. Not sure what the current process is for releasing to Live, do push any changes to this branch that would reflect the current process better 😀

This looks good @henocookie, the release to live steps also look correct.

File: README.md [hunk 1]
```diff
@@ -20,9 +20,9 @@ Run the project in development mode `npm run watch` and visit <a href="http://lo
 
 ### Environments
 
-- [Review](https://nhsuk-service-manual-pr.azurewebsites.net/) - code pushed to the `review` branch will deploy to the Review environment (optional environment for content review and development)
-- [Staging](https://nhsuk-service-manual-dev-uks.azurewebsites.net/) - code pushed to the `master` branch will deploy to the Staging environment
-- [Live](https://service-manual.nhs.uk/) - git tags created using SEMVER format `v0.0.0` will deploy to the Live environment. (Note: Live is behind a server side cache which results in changes not appearing immediately.)
+- Review - used to review content, design, development and research changes. Code pushed on a branch starting with `review/0-` will deploy to the [review/0 environment](https://nhsuk-service-manual-pr.azurewebsites.net/). Code pushed on a branch starting with `review/1-` will deploy to the [review/1 environment](http://nhsuk-service-manual-pr-nhsuk-service-manual-r1.azurewebsites.net/).
```
Comment (DomBaker) [inline README.md:23]: I'd probably swap the "to" and "on", we push to a branch and deploy on a review

**Review by DomBaker — APPROVED** [review]
Reviewed and approved.

**Pull request merged**